js/repl/ReplOptions.tsx

```diff
@@ -465,12 +456,10 @@ class ExpandedContainer extends Component<Props, State> {
                     "decoratorsVersion",
                     (t) => t.value
                   )}
+                  value={presetsOptions.decoratorsVersion}
                 >
                   {Object.keys(DECORATOR_PROPOSALS).map((key) => (
-                    <option
-                      value={key}
-                      selected={key === presetsOptions.decoratorsVersion}
-                    >
```

Comment:
Doesn't removing `selected` change the behavior?

Reply:
In react 18, passing `selected` to `option` is not supported: https://react.dev/reference/react-dom/components/select#caveats. I didn't check, maybe it was implemented in an earlier React version before.